app/src/main/java/com/example/peterchu/watplanner/Models/Course.java

```diff
@@ -18,6 +19,7 @@
     private String title;
 
     public Course(String subject, String number, String credits, String title) {
+        this.id = -1;
```

Comment:
Retrofit automatically converts the Course objects in the JSON directly to Course objects in Java. The arguments have to match (and id of course woudln't exist in the JSON returned by the API)